roles/pre_tasks/templates/inventory.j2

```diff
@@ -9,6 +9,12 @@
 [all:vars]
 admin_password='{{ tower_admin_password }}'
 
+{% if tower_ssh_connection_vars is defined %}
+{% for item in tower_ssh_connection_vars %}
+{{ item.name}}='{{ item.value }}'
```

Comment:
This is presumably a lint error. The linting failed anyway but looks like for another reason.

Reply:
@Tompage1994 I've been hunting down trying to figure this one out, if anyone has any ideas I'd be grateful.

Reply:
@Tompage1994 Tracked it down, wasn't a linting issue, it was a build issue in our linting precheck system, switched over to the precommit system we use in tower_config and added a few issues, Not sure on role names, but I believe since our galaxy.yml is variablized it doesn't pick up the role names correctly, Also on other tasks it expects chmod to be set, we will need to test that, but created an issue for it.